mula/scheduler/context/context.py

```diff
@@ -103,3 +103,9 @@ def __init__(self) -> None:
             registry=self.metrics_registry,
             labelnames=["scheduler_id"],
         )
+
+    def _remove_trailing_slash(self, url: str) -> str:
```

Comment:
Two things:
- This is a more common "problem" and I'd expect this in some kind of utils module rather than this class specifically
- It could've been a (static) function